Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/viewmodel/SubmissionViewModel.kt

```diff
@@ -112,6 +113,11 @@ class SubmissionViewModel : ViewModel() {
         SubmissionService.deleteTestGUID()
     }
 
+    fun submitWithNoDiagnosisKeys() {
+        BackgroundWorkScheduler.stopWorkScheduler()
+        LocalData.numberOfSuccessfulSubmissions(1)
```

Comment:
The SubmissionViewModel already has the SubmissionService as a dependency. Instead of duplicating the code here, we can call `SubmissionService.submissionSuccessful()`

Reply:
Had a word with Kolya yesterday, due to future changes such as DI, it was suggested that the method should be in ViewModel. I am aware of the fact that it's repeating, @kolyaopahle  what do you think? I can just call the SubmissionService.submissionSuccessful() function in Viewmodel but may not be a good idea to have inter-dependencies in ViewModel and submissionService. 

let me know what you think

Reply:
I think the original comment referred to not adding an additional dependency to the Fragment. The ViewModel already has a dependency to SubmissionService, therefore I see no issue with this.